profiler/README.md

```diff
@@ -15,3 +52,84 @@ It should be considered experimental and is completely unsupported.
 |`splunk.profiler.period.{eventName}`      | n/a                    | customize period (in ms) for a specific jfr event. For example, to set the ThreadDump frequency to 1s (1000ms): `-Dsplunk.profiler.period.threaddump=1000` |
 |`splunk.profiler.tlab.enabled`            | false                  | set to `false` to disable TLAB memory events |
 |`splunk.profiler.include.agent.internals` | false                  | set to `true` to include agent internal call stacks |
+
+# Escape hatch
+
+The profiler limits its own behavior under two conditions:
+
+* Free disk space is too low (< 100MB free))
+* More than 5 minutes worth of JFR files are backed up on disk
+
+If a recording is already running when this condition is encountered, it will
+be stopped. Later, when the conditions are no longer applicable, then recording will
+be restarted and the profiler will resume operation.
+
+# FAQ / Troubleshooting
+
+### How do I know if it's working?
+
+At startup, the agent will log the string "JFR profiler is active" at `INFO`. You can grep for this in your logs to see
+something like this:
+```
+[otel.javaagent 2021-09-28 18:17:04:246 +0000] [main] INFO com.splunk.opentelemetry.profiler.JfrActivator - JFR profiler is active.
+```
```

Comment:
```suggestion
### Check that the profiler is working

If the profiler is enabled, the agent logs the string "JFR profiler is active" as an `INFO` entry at startup. You can grep your logs to find entries similar to the following:
```
[otel.javaagent 2021-09-28 18:17:04:246 +0000] [main] INFO com.splunk.opentelemetry.profiler.JfrActivator - JFR profiler is active.
```
```